ghcl/cache.py

```diff
@@ -0,0 +1,28 @@
+import redis
```

Comment:
Can you run `pipenv install redis` on your machine and commit the `Pipfile` and `Pipfile.lock` as well please?